### ISSUE TITLE
The winform program written in C# uses ContextMenuStrip to display menus in which there are submenus, and NVDA cannot read 

### ISSUE THREAD

**Issue by c469591** [nvaccess/nvda#14863]
### Is your feature request related to a problem? Please describe.

Yes.
The winform program written in C# uses ContextMenuStrip to display menus in which there are submenus, and NVDA cannot read aloud whether there are submenus or not.
The native screen reader software in windows can read aloud.
ZDSR, another screen reader, also reads aloud.
The software used is the OCR recognition software developed by a blind Chinese.
Computer version download link →
http://api.lmdbk.com/dl/%E8%90%BD%E5%AF%9E%E5%B7%A5%E5%85%B7%E7%AE%B1%E7%94%B5%E8%84%91%E5%AE%89%E8%A3%85%E6%9D%BF%EF%BC%8C%E8%BE%93%E5%85%A5%E5%AF%86%E7%A0%81lm%E8%BF%9B%E8%A1%8C%E8%A7%A3%E5%8E%8B.7z
Unzip the password →
lm
After opening the software, press
win+alt+p to open the submenu, but NVDA are unable to read aloud the submenu
For example, other screen guide software will read aloud
Settings submenu
while NVDA will only read aloud
Settings
I am sorry, because my native language is not English, so some words may be wrong

### Describe the solution you'd like

讓NVDA可以朗讀子菜單

### Describe alternatives you've considered

I don't know

### Additional context

As mentioned above
